File: 백준/Silver/1427. 소트인사이드/소트인사이드.java
import java.io.*;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringBuilder sb = new StringBuilder();

        long n = Long.parseLong(br.readLine());
        String s = "" + n;
        String arr[] = s.split("");

        Arrays.sort(arr);

        for(int i = arr.length-1; i >= 0; i--){
            sb.append(Long.parseLong(arr[i]));
        }

        bw.write(sb.toString());
        bw.flush();
    }
}

File: 백준/Silver/1427. 소트인사이드/소트인사이드.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val bw = BufferedWriter(OutputStreamWriter(System.out))
    val sb = StringBuilder()

    val input = br.readLine()
    var arr = input.toCharArray().sortedDescending()

    for (num in arr) {
        sb.append(num)
    }

    bw.write(sb.toString())
    bw.flush()
    bw.close()
}